enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/OCINode.java

```diff
@@ -124,10 +129,12 @@ public Node.Handle getHandle() {
     private final class RefreshListener implements PropertyChangeListener {
         @Override
         public void propertyChange(PropertyChangeEvent evt) {
-            if ("referenceName".equals(evt.getPropertyName())) {
+            if ("referenceName".equals(evt.getPropertyName()) //NOI18N
+                    || "publicIp".equals(evt.getPropertyName())) { //NOI18N
```

Comment:
Now where there's fireDisplayNameChange in both branches - would it be too bad to call `refresh` + `fireDisplayNameChange` for all cases ?

Reply:
It would reload children even in case only Node's display name needs to be updated.